pkg/asset/machines/worker.go

```diff
@@ -94,6 +94,11 @@ func (w *Worker) Generate(dependencies asset.Parents) error {
 	switch ic.Platform.Name() {
 	case awstypes.Name:
 		mpool := defaultAWSMachinePoolPlatform()
+		// worker pool default volume settings
+		mpool.EC2RootVolume = awstypes.EC2RootVolume{
```

Comment:
nit: can we move this to defaultAWSMachinePoolPlatform and only override the `Size` in master.

Reply:
done, thanks!